docs/onnx.md

```diff
@@ -0,0 +1,55 @@
+---
+title: ONNX model inferencing on Spark
+description: Learn how to use the ONNX model transformer to run inference for an ONNX model on Spark.
+---
+
+# ONNX model inferencing on Spark
+
+## ONNX
+
+[ONNX](https://onnx.ai/) is an open format to represent both deep learning and traditional machine learning models. With ONNX, AI developers can more easily move models between state-of-the-art tools and choose the combination that is best for them.
+
+MMLSpark now includes a Spark transformer to bring an trained ONNX model to Apache Spark, so you can run inference on your data with Spark's large-scale data processing power.
+
+## Setup and installation
+
+In addition to the mmlspark-core and mmlspark-deep-learning jars, the ONNX model transformer also depends on the ONNX runtime library.
+
+For CPU clusters, the maven coordinate is `com.microsoft.onnxruntime:onnxruntime:1.8.1` and for GPU clusters, the maven coordinate is `com.microsoft.onnxruntime:onnxruntime_gpu:1.8.1`.
```

Comment:
I thought this was included in the deps?

Reply:
But they're not packaged in the mmlspark jars right? So in Databricks clusters, I always had to manually install external maven dependencies like `com.microsoft.cntk:cntk:2.4`, `com.microsoft.ml.lightgbm:lightgbmlib`, `io.spray:spray-json:1.3.2` and `org.openpnp:opencv:3.2.0-1`. In this case, you can install either "onnxruntime" or "onnxruntime_gpu". The APIs are exactly the same, but the "onnxruntime" jar doesn't have CUDA dependencies packaged together, so won't be able to utilize GPU, and will fallback to use CPU on a GPU cluster.

Reply:
Turns out Databricks installs the transitive dependencies. I was under the impression it wouldn't do so. Anyway I removed this section from the doc. It no longer applies.